internal/execute/testsys_test.go

```diff
@@ -152,6 +152,18 @@ func (s *testSys) Writer() io.Writer {
 	return s.currentWrite
 }
 
+func (s *testSys) WriteOutputIsTTY() bool {
+	return true
+}
+
+func (s *testSys) GetWidthOfTerminal() int {
+	return 0
```

Comment:
This one was implemented as TS_TEST_TERMINAL_WIDTH environment variable value. do same or something similar